bindings/java/wasmedge-java/wasmedge-jni/lib/GlobalTypeContext.c

```diff
@@ -12,8 +12,35 @@ GETTER(GlobalTypeContext)
 JNIEXPORT void JNICALL Java_org_wasmedge_GlobalTypeContext_nativeInit(
     JNIEnv *env, jobject thisObject, jint valueType, jint mutability) {
 
+  // valueType conversion
+  uint8_t type = valueType;
+  WasmEdge_ValType finalType;
+  switch(type){
+    case 0x7F:
+        finalType = WasmEdge_ValTypeGenI32();
+        break;
+    case 0x7E:
+        finalType = WasmEdge_ValTypeGenI64();
+        break;
+    case 0x7D:
+        finalType = WasmEdge_ValTypeGenF32();
+        break;
+    case 0x7C:
+        finalType = WasmEdge_ValTypeGenF64();
+        break;
+    case 0x7B:
+        finalType = WasmEdge_ValTypeGenV128();
+        break;
+    case 0x70:
+        finalType = WasmEdge_ValTypeGenFuncRef();
+        break;
+    case 0x6F:
+        finalType = WasmEdge_ValTypeGenExternRef();
+        break;
+    }
+
```

Comment:
@q82419 I'm wondering does there exist a more automatic solution? Since component model will put tons of repeated work here.